app/components/progress-donut.ts

```diff
@@ -8,6 +8,7 @@ interface Signature {
   Args: {
     total: number;
     completed: number;
+    variant?: 'default' | 'teal' | 'grey';
```

Comment:
What's your thought process behind this name? Why did we choose `variant` instead of `color`?